specification/metrics/semantic_conventions/runtime-environment-metrics.md

```diff
@@ -297,6 +298,22 @@ This metric is obtained from [`ClassLoadingMXBean#getLoadedClassCount()`](https:
 <!-- semconv metric.process.runtime.jvm.classes.current_loaded(full) -->
 <!-- endsemconv -->
 
+### Metric: `process.runtime.jvm.cpu.time`
+
+This metric is [recommended][MetricRecommended].
+
+This metric is obtained from [`com.sun.management.OperatingSystemMXBean#getProcessCpuTime()`](https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getProcessCpuTime()) on HotSpot
```

Comment:
Wouldn't it be nicer if these are added as `note` in `yaml`, as done in https://github.com/open-telemetry/semantic-conventions/pull/13?

Reply:
I just noticed that `notes` for metrics are not being generated in the markdown files. They only seem to show up for `attributes`. I will investigate the build tools and open an issue if that's the case.